src/lib/components/events/event-item.svelte

```diff
@@ -57,16 +64,18 @@
         href={info.meetingLink}
         role="button"
         target="_blank"
-        rel="noopener noreferrer">Join</a
+        rel="noopener noreferrer"
+        on:click={(/* janky hack */) => {
+          shown = !shown;
+        }}>Join</a
       >
     </summary>
 
-    <hr />
-
-    <p class="event-description">
-      {@html info.description}
-    </p>
-
+    {#if shown}
```

Comment:
This breaks without JavaScript. We should keep a `<noscript>` tag outside of the `if` block like so:

```html
<noscript>
  <p class="event-description">
    {@html info.description}
  </p>
</noscript>
{#if shown}
  <!-- same p tag as above -->
{/if}
```